www/source/partials/docs/examples/kubernetes_hab_bastion/plan.sh

```diff
@@ -0,0 +1,16 @@
+pkg_name=hab_bastion
+pkg_origin=habitat
+pkg_version="0.1.0"
+pkg_maintainer="irvingpop"
+pkg_license=("Apache-2.0")
+pkg_deps=(core/busybox-static)
+pkg_svc_run="while true; do sleep 60; done"
+
+do_build() {
+  return 0
+}
+
+do_install() {
+  return 0
+}
+
```

Comment:
We _really_ need a formally-supported way to get a container image that _just_ has a Supervisor on it, rather than having to create a "dummy" service to run.

We have at least one running around for some tests, but this should probably be a release artifact.

This shouldn't hold up publication of this documentation; it's more of a reminder to me to schedule this work :sweat_smile:

Reply:
:boom: https://github.com/habitat-sh/habitat/issues/7161